### PR TITLE
Jose/update2.1.13

### PR DESCRIPTION
**Pull request by jotathebest**

### CODE REVIEW

File: src/Ubidots.cpp [hunk 1]
```diff
@@ -159,98 +146,85 @@ float Ubidots::getValue(char* id) {
...
+  }
 
-    response = response.substring(value_init);
-    if (_debug) {
-        Serial.println("Value obtained:");
-        Serial.println(response);
+  timeout = 0;
+  free(data);
+
+  while(!_client.available() && timeout < 50000) {
```
Comment (woakas) [inline src/Ubidots.cpp]: Move this values to a constants

File: src/Ubidots.cpp [hunk 2]
```diff
@@ -262,100 +236,88 @@ float Ubidots::getValueWithDatasource(char* device, char* variable) {
...
+  }
+
+  if (_debug) {
+    Serial.println(F("Getting your variable with request: "));
+    Serial.println(F(data));
+  }
+
+  _client.print(data);
+  timeout = 0;
+
```
Comment (woakas) [inline src/Ubidots.cpp]: Missing a free(data)

File: src/Ubidots.cpp (hunk 1 above)
Comment (jotathebest) [reply src/Ubidots.cpp]: fixed

File: src/Ubidots.cpp (hunk 2 above)
Comment (jotathebest) [reply src/Ubidots.cpp]: fixed

**Pull request merged**